docs/attachments.md

```diff
@@ -92,3 +141,55 @@ Following is a sample proxy service that illustrates how to connect to Salesforc
       <description/>
   </proxy>
 ```
+2. Create a xml file called createJob.xml containing the following xml:
```

Comment:
Create an XML file named createJob.xml and copy the configurations given below to it:

Reply:
Fixed in a0448a990213d3cdb3f3ae158ef2307753bad900.